ddl/serial_test.go

```diff
@@ -68,6 +82,205 @@ func (s *testSerialSuite) TearDownSuite(c *C) {
 	}
 }
 
+func (s *testSerialSuite) TestPrimaryKey(c *C) {
```

Comment:
The new tests for this file come from db_integration_test.go because they require `newCfg.EnablePKIsHandle = true`.

Reply:
got it.